cas-client-core/src/main/java/org/jasig/cas/client/configuration/PropertiesConfigurationStrategyImpl.java

```diff
@@ -86,16 +94,28 @@ private boolean loadPropertiesFromFile(final String file) {
         if (CommonUtils.isEmpty(file)) {
             return false;
         }
-        FileInputStream fis = null;
-        try {
-            fis = new FileInputStream(file);
-            this.properties.load(fis);
-            return true;
-        } catch (final IOException e) {
-            LOGGER.warn("Unable to load properties for file {}", file, e);
-            return false;
-        } finally {
-            CommonUtils.closeQuietly(fis);
-        }
+
+	if (file.startsWith(CLASSPATH_PREFIX)) {
```

Comment:
we might be able to simplify this by doing the InputStream assignment in its own if/else and then just doing the properties.load stuff first?